src/SignalRServiceExtension/Config/IServiceHubContextStore.cs

```diff
@@ -6,8 +6,10 @@
 
 namespace Microsoft.Azure.WebJobs.Extensions.SignalRService
 {
-    internal interface IServiceHubContextStore
+    public interface IServiceHubContextStore
     {
         ValueTask<IServiceHubContext> GetOrAddAsync(string hubName);
+
+        IServiceManager ServiceManager { get; set; }
```

Comment:
No need to set?

Reply:
I think it's necessary. It can be used to crate access token. For example
```
        [FunctionName("negotiate")]
        public static SignalRConnectionInfo GetSignalRInfo(
            [HttpTrigger(AuthorizationLevel.Anonymous)] HttpRequest req,
            [SignalRConnectionInfo(HubName = "simplechat", UserId = "{headers.x-ms-signalr-userid}")] SignalRConnectionInfo connectionInfo)
        {
            var accessToken = StaticServiceManagerStore.Get().ServiceManager
                .GenerateClientAccessToken(
                    "simplechat",
                    req.Query["userid"],
                    new List<Claim> { new Claim("claimType", "claimValue") });
            connectionInfo.AccessToken = accessToken;
            return connectionInfo;
        }
```

I will put it in the sample.

Reply:
but getter should be enough?